torch/nn/_functions/loss.py

```diff
@@ -167,7 +167,7 @@ def forward(ctx, input1, input2, y, margin, size_average):
             output = output / y.size(0)
 
         ctx.save_for_backward(input1, input2, y)
-        return input1.new((output,))
```

Comment:
do we support scalars now?

Reply:
Not really -- scalars are conditionally enabled by a build flag (`WITH_SCALARS=1`) for development.

Reply:
So how does this work now? Isn't output a Python scalar in here?

Reply:
It's a 1-element 1-d tensor.

Reply:
ohhh because those are Variables so they never degenerate into scalars. makes sense